third_party/java/java_tools/BUILD

```diff
@@ -37,6 +37,6 @@ genrule(
         "//third_party/java/jdk/langtools:jdk_compiler_jar",
     ],
     outs = ["java_tools.zip"],
-    cmd = "zip -j $(OUTS) $(SRCS)",
+    cmd = "zip -j $@ cat $(SRCS) | sort",
```

Comment:
Did you mean:
```
cmd = "zip -j $@ $$(echo $(SRCS) | sort)",
```

Reply:
Uhm some leftover code that I forgot to update last Friday. Thanks for spotting!